attachments/static/attachments/js/attachments.js

```diff
@@ -54,6 +68,16 @@
             else {
                 if(settings.error) {
                     settings.error(data);
+                    if (settings.enableRetry) {
+                        if (!$(settings.attachmentsRetryButtonID).length) {
+                            $(settings.container).append(
+                                '<button type="button" id="' + settings.attachmentsRetryButtonID + '">Retry</button>'
+                            );
+                        }
+                        if (settings.attachmentsRetryButtonID.length) {
```

Comment:
Do we need this if statement? If the button doesn't exist you're adding it above.  By the time you get here I think it's safe to say the button will exist

Reply:
True -- updated